sql/mysqld.cc

```diff
@@ -3544,6 +3544,9 @@ SHOW_VAR com_status_vars[] = {
     {"purge_before_date",
      (char *)offsetof(System_status_var, com_stat[(uint)SQLCOM_PURGE_BEFORE]),
      SHOW_LONG_STATUS, SHOW_SCOPE_ALL},
+    {"purge_gtid",
+     (char *)offsetof(System_status_var, com_stat[(uint)SQLCOM_PURGE_UUID]),
+     SHOW_LONG_STATUS},
```

Comment:
Also missing SHOW_SCOPE_ALL. I'll fix this before pushing.

Reply:
Fixed.